main/platform/README.md

```diff
@@ -0,0 +1,64 @@
+# The Agoric Platform
+
+![agoric stack](./assets/stack.svg "The Agoric Stack")
+
+This document focuses on the layers beneath Zoe and ERTP, what we call
+the Agoric Platform. This includes "Swingset", which can be thought of
```

Comment:
In the top diagram, it's "SwingSet". Here, it's "Swingset". Pretty sure I recall seeing "SwingSet" other places as well. Should decide on a consistent capitalization.

Reply:
I agree
I'm creating a follow-up issue so that this concern does not block merging this content : https://github.com/Agoric/documentation/issues/163